serapi/serapi_goals.ml

```diff
@@ -74,7 +78,7 @@ let process_goal_gen ppx sigma g : 'a reified_goal =
   let ppx       = ppx env sigma                                             in
   let hyp       = List.map (get_hyp ppx sigma) ctx                          in
   let info      = build_info sigma g                                        in
-  { info; ty = get_goal_type ppx sigma g; hyp }
+  { info; ty = get_goal_type ppx env sigma g; hyp }
```

Comment:
Isn't this env new compared to the sigma?

Reply:
WDYM? It's an environment in which the evar type makes sense.

Reply:
oh yes nvm I confused myself.